app/src/main/java/com/readrops/app/activities/ItemActivity.java

```diff
@@ -278,23 +282,48 @@ public void onCreateContextMenu(ContextMenu menu, View v, ContextMenu.ContextMen
 
         if (hitTestResult.getType() == WebView.HitTestResult.IMAGE_TYPE ||
                 hitTestResult.getType() == WebView.HitTestResult.SRC_IMAGE_ANCHOR_TYPE) {
-            new MaterialDialog.Builder(this)
-                    .title(R.string.image_options)
-                    .items(R.array.image_options)
-                    .itemsCallback((dialog, itemView, position, text) -> {
-                        if (position == 0)
-                            shareImage(hitTestResult.getExtra());
+            MaterialDialog.Builder builder = new MaterialDialog.Builder(this);
```

Comment:
Can you make chained calls with the Material Dialog builder like this: 
```java
new MaterialDialog.Builder(this)
          .title(...)
          .items(...)
          .itemsCallback((dialog, itemView, position, text) -> {

          })
         .show();
```
It is easier to read and makes a better use of the builder.